client/views/actions/whois.tpl

```diff
@@ -1,93 +1,79 @@
-<div>
-	{{> ../user_name nick=whois.nick}}
-	<i class="hostmask">({{whois.user}}@{{whois.host}})</i>
-</div>
+<p>{{> ../user_name nick=whois.nick}}</p>
+<dl class="whois">
+	<dt>Host mask</dt>
+	<dd class="hostmask">{{whois.user}}@{{whois.host}}</dd>
+
 {{#if whois.actualhost}}
-<div>
-	{{> ../user_name nick=whois.nick}}'s actual host is: <i>{{whois.actualhost}}</i> (<i>{{whois.actualip}}</i>)
-</div>
+	<dt>Actual host</dt>
+	<dd class="hostmask"><a href="https://ipinfo.io/{{whois.actualip}}" target="_blank" rel="noopener">{{whois.actualip}}</a>{{#equal whois.actualhost whois.actualip}}{{else}} ({{whois.actualhost}}){{/equal}}</dd>
 {{/if}}
+
 {{#if whois.real_name}}
-<div>
-	{{> ../user_name nick=whois.nick}}'s real name is:
-	<b>{{{parse whois.real_name}}}</b>
-</div>
+	<dt>Real name</dt>
+	<dd>{{{parse whois.real_name}}}</dd>
 {{/if}}
+
 {{#if whois.account}}
-<div>
-	{{> ../user_name nick=whois.nick}}
-	is logged in as <b>{{whois.account}}</b>
-</div>
+	<dt>Logged in as</dt>
+	<dd>{{whois.account}}</dd>
 {{/if}}
+
 {{#if whois.registered_nick}}
-<div>
-	{{> ../user_name nick=whois.nick}}
-	{{whois.registered_nick}}
-</div>
+	<dt>Registered nick</dt>
+	<dd>{{whois.registered_nick}}</dd>
 {{/if}}
+
 {{#if whois.modes}}
-<div>
-	{{> ../user_name nick=whois.nick}}
-	{{whois.modes}}
-</div>
+	<dt>Modes</dt>
+	<dd>{{whois.modes}}</dd>
 {{/if}}
+
 {{#if whois.special}}
-<div>
-	{{> ../user_name nick=whois.nick}}
-	{{whois.special}}
-</div>
+	<dt>Special</dt>
+	<dd>{{whois.special}}</dd>
 {{/if}}
+
 {{#if whois.operator}}
-<div>
-	{{> ../user_name nick=whois.nick}}
-	{{whois.operator}}
-</div>
+	<dt>Operator</dt>
+	<dd>{{whois.operator}}</dd>
```

Comment:
Is this a boolean or a text value?

Reply:
Afaik it is a text value, you can actually see what is and isn't by looking at the current whois.tpl. When I did the previous PR for that I translated booleans to something userfriendly will directly showing anything else.